src/lakefs_spec/spec.py

```diff
@@ -441,31 +447,95 @@ def _open(
             **kwargs,
         )
 
+    def put_file_to_blockstore(
+        self, lpath, repository, branch, resource, presign=True, storage_options={}
+    ):
+        print("In Blockstore")
+        if tuple(int(v) for v in __lakefs_client_version__.split(".")) < (0, 111, 0):
+            blockstore_type = self.client.config_api.get_storage_config().blockstore_type
+        else:
+            print("OOOKS")
+            print(self.client.config_api.get_config())
+            print(self.client.config_api.get_config().storage_config)
+            blockstore_type = self.client.config_api.get_config().storage_config.blockstore_type
+            print(blockstore_type)
+        if blockstore_type == "local":
+            raise ValueError(
+                "Cannot write to blockstore of type 'local'. Disable use_blockstore or configure remote blockstore."
+            )
```

Comment:
1. Remove debug prints please,
2. You can use `from lakefs_spec import lakefs_client_version` instead of the split in L454,
3. I would change the errormsg to something shorter, like e.g. `block store writes not implemented for block store type 'local'`.

Reply:
Importing throws: `ImportError: cannot import name 'lakefs_client_version' from partially initialized module 'lakefs_spec' (most likely due to a circular import)`

Reply:
As for the print statements that slipped through. Should we add a test for print statements to the `pre-commit` checks?